README.md

```diff
@@ -263,6 +264,7 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
    * https://apiary.io/ - Collaborative design API with instant API mock and generated documentation (Free for unlimited API blueprints and unlimited user with one admin account and hosted documentation)
    * https://www.jetbrains.com/products.html - Productivity tools, IDEs and deploy tools. Free license for students, teachers, open source projects, and user groups.
    * https://readme.io/ - Beautiful documentations made easy - free for Open Source
+   * http://runnable.com/ - IDE and terminal in a browser, good place to share your code, for free.
```

Comment:
this is a beta service and they dont seem to mention their long term plans or indicate that whats free now will remain free.

From the original blog post announcing this project:

```
I am not after services in Private Beta or Free During Beta or things like this, the only ones of those I’d accept are ones who specifically state that beta accounts will become free dev accounts or similar in future.
```

So gonna close this one sorry